README.md

```diff
@@ -32,7 +32,8 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
-The application uses Trello's API to fetch and save to-do tasks. In order to call their API, you need to ﬁrst [create an account](http://trello.com/signup), then generate an API key and token by following the [instructions here](https://trello.com/app-key). TRELLO_KEY and TRELLO_TOKEN variables need to be set in the `.env` file.
+The application uses MongoDB to fetch and save to-do tasks. DB_CONNECTION_URL, TODO_DB_NAME and TODO_BOARD_ID variables need to be set in the `.env` file.
```

Comment:
Not sure I've understood exactly why you have a TODO_BOARD_ID - is there any reason to assume you would want multiple boards inside the same DB? You always use the same `cards` collection (which makes sense) so it feels to me like you could just use the DB_NAME to differentiate between e.g. environments?
Not a big issue at all, just worth considering!